src/test/java/com/ozonehis/eip/routes/senaite/CreateAnalysisRequestToSenaiteRouteITest.java

```diff
@@ -34,25 +26,25 @@ public class CreateAnalysisRequestToSenaiteRouteITest extends BaseWatcherRouteTe
     @EndpointInject(value = "mock:createAnalysisRequestSenaiteEndpoint")
     private MockEndpoint createAnalysisRequestSenaiteEndpoint;
     
-    @Before
+    @BeforeEach
     public void setup() throws Exception {
-    	loadXmlRoutesInDirectory("senaite", "create-analysisrequest-to-senaite-route.xml");
-    	RouteDefinition routeDefinition = camelContext.adapt(ModelCamelContext.class).getRouteDefinitions().stream().filter(routeDef -> "create-analysisrequest-to-senaite".equals(routeDef.getRouteId())).collect(Collectors.toList()).get(0);
-    	RouteReifier.adviceWith(routeDefinition, camelContext, new AdviceWithRouteBuilder() {
+    	loadXmlRoutesInDirectory("camel", "create-analysisrequest-to-senaite-route.xml");
+    
+    	advise("create-analysisrequest-to-senaite", new AdviceWithRouteBuilder() {
     	    @Override
-    	    public void configure() throws Exception {
+    	    public void configure() {
     	    	weaveByToString("To[direct:authenticate-to-senaite]").replace().toD("mock:authenticateToSenaiteRoute");
-    	    	weaveByToString("DynamicTo[{{senaite.baseUrl}}/@@API/senaite/v1/search?getClientSampleID=${exchangeProperty.lab-order-uuid}&getClientID=${exchangeProperty.patient-id}&catalog=senaite_catalog_sample&complete=true]").replace().toD("mock:searchAnalysisRequestSenaiteEndpoint");
```

Comment:
Is this `weaveByToString("DynamicTo[...]` syntax no longer supported? The wild card replacement is a bit more ambiguous and unclear in some places.

Reply:
`weaveByToString(...)` is still supported. I opted for wildcard because when `{{senaite.baseUrl}}` resolves different values while running all tests. The reasons for this;
- Test setup isn't wired correctly
- Inherits from parent `openmrs-eip:commons:test-jar` which makes modification a bit tricky